### PR TITLE
[DOC] Improve FBX Prefab Variants section

### PR DESCRIPTION
**Pull request by sebastienduverne**
### Purpose of this PR

[FBX-40](https://jira.unity3d.com/browse/FBX-40) - Improve Prefab Variants section in documentation

Fully rewriting (almost from scratch) the whole section about how and why to use the "Convert To FBX Prefab Variant" option:
- [x] Added a landing page "Work with FBX Prefab Variants".
- [x] Added a concept page to summarize in one place the basics to know about FBX files, Model Prefabs, Prefab Variants, and what we call FBX Prefab Variants in the FBX Exporter package.
- [x] Added a concept page targeting the benefits and limitations to set up an FBX Prefab Variant workflow.
- [x] Added 3 task pages to cover the conversion of a GameObject, a Prefab asset, and a Model Prefab to an FBX-based Prefab Variant.
- [x] Removed obsolete content about the former concept of Linked Prefabs.

### Testing

- [x] Successfully tested the documentation build using Package Doc Tools.

### Note for reviewers

To ease the review process, a built version of the updated doc is [available in Google Drive](https://drive.google.com/drive/folders/1LDepwsWmNClMu72KiKJ68o6MQMtnTAj2?usp=drive_link). To read it, download the `com.unity.formats.fbx@5.1.zip` file, unzip it, and open the `index.html` file in the root folder.

Note that this PR targets the _Work with FBX Prefab Variants_ section only. Further improvements are planned for the rest of the documentation.

### CODE REVIEW

**Review by mfe — APPROVED** [review]
I checked the links and read all the sections. Everything was clear.
That's a great update, thanks Seb !

File: com.unity.formats.fbx/Documentation~/prefab-variants-workflow.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* Override some properties of the Prefab Variants without affecting the FBX files they're based on.
+
+For example, if a Model Prefab includes a Spot Light of size 10 and you override the size to 1 in the Prefab Variant, when the size and color change in the source FBX file, the color changes in the Prefab Variant but the size remains 1.
+
+>[!WARNING]
+>To avoid any data conflicts and unexpected behaviors between the Prefab Variant and the base FBX file, you must not make structural changes to the Prefab Variant. This includes deleting, reordering, and renaming inherited child GameObjects.
+
+## Available features
+
+The FBX Exporter package allows to [convert a GameObject and its children](prefab-variants-convert-gameobject.md), or [a Prefab asset](prefab-variants-convert-prefab-asset.md), to an FBX Prefab Variant. Such a conversion corresponds to an FBX export followed by a Prefab Variant creation, all in one single action.
```
Comment (chloe-azoto) [inline com.unity.formats.fbx/Documentation~/prefab-variants-workflow.md:20]:
```suggestion
The FBX Exporter package allows you to [convert a GameObject and its children](prefab-variants-convert-gameobject.md), or to convert [a Prefab asset](prefab-variants-convert-prefab-asset.md) to an FBX Prefab Variant. Such a conversion corresponds to an FBX export followed by a Prefab Variant creation, all in one single action.
```

File: com.unity.formats.fbx/Documentation~/prefab-variants-concepts.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### Model Prefab management
+
+You can adjust the way Unity handles the Model Prefab created from an FBX file by [changing the import settings](https://docs.unity3d.com/Documentation/Manual/class-FBXImporter.html), but you can't edit the Model Prefab with the Unity Editor, contrary to a Unity Prefab.
+
+You can instantiate a Model Prefab in a Scene, but a [best practice](prefab-variants-workflow.md) is to first create a Prefab Variant based on the Model Prefab and then instantiate the created Prefab Variant instead of the Model Prefab.
+
+### FBX file edit
+
+You can open an FBX file stored in your Unity project using a 3D modeling application such as Autodesk® Maya®. Once you edit and save the FBX file via the 3D modeling application, Unity automatically updates the corresponding Model Prefab in your project.
```
[the remark this answers is not in the document]
Comment (sebastienduverne) [reply com.unity.formats.fbx/Documentation~/prefab-variants-concepts.md:21]: Actually, there is a link to the Workflow section at the bottom of the page in "Additional resources" (and it was pointing to the wrong place, so I fixed it!) – so I'm not overlinking to it, and I even removed the one from the previous paragraph.

File: com.unity.formats.fbx/Documentation~/prefab-variants-convert-gameobject.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Further management
+
+After you convert the GameObject:
+
+* The exported FBX file becomes the model source which you can edit via a 3D modeling application.
+
+* The created Prefab Variant becomes the only asset you should manipulate and instantiate in your Unity project.
+
+* Any original Prefab or Model Prefab that were instantiated as children of the converted GameObject have no more relationship with the created FBX file and Prefab Variant. You might get rid of these original assets unless you need them for another context.
```
[the remark this answers is not in the document]
Comment (sebastienduverne) [reply com.unity.formats.fbx/Documentation~/prefab-variants-convert-gameobject.md:33]: I agree it's not so clear. So, everything that was selected for conversion in the Scene is replaced by the created Prefab Variant. I'm talking about the Prefab assets (in Project) that had instances part of the selection.
I'm trying to provide a clearer version of this paragraph.

**Review by lucillecaillaud — APPROVED** [review]
I'm good with the latest change!
I think it's more clear and more precise! Thanks!

**Pull request merged**